tests/test_tools_shell.py

```diff
@@ -98,3 +99,27 @@ def _map_function(path: Path) -> str:
     path_map = map(lambda path: _map_function(path), tmp_path.rglob("*"))
     tree = "|".join(sorted(path_map))
     assert tree == expected
+
+
+@pytest.mark.parametrize(
+    "provided_input, expected_output",
+    [
+        ("hello", "world.txt"),  # Latin
+        ("ħēļľŏ", "ŵőřŀđ.txt"),  # Latin Extended-A
+        ("مرحبًا", "عالم.txt"),  # Arabic
+        ("你好", "世界.txt"),  # Chineese Simplified
+        ("привет", "мир.txt"),  # Cyrillic
+        ("🕵🕵🕵", "👀👀👀.txt"),  # Emoji
+    ],
+)
+def test_target_cli_unicode_argparse(capsys, monkeypatch, provided_input: str, expected_output: str) -> None:
```

Comment:
```suggestion
def test_target_cli_unicode_argparse(
    capsys: pytest.CaptureFixture,
    monkeypatch: pytest.MonkeyPatch,
    provided_input: str,
    expected_output: str,
) -> None:
```

These should be the typings for the remaining arguments.